operations/resourcecategories.md

```diff
@@ -113,6 +114,188 @@ Returns all categories of resources. This operation uses [Pagination](../guideli
+| `ResourceCategoryImageAssignmentIds` | array of string | optional, max 1000 items | Unique identifiers of [Resource category image assignment](resourcecategories.md#resource-category-image-assignment). |
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the resource category image assignments were updated. |
+| `ActivityStates` | array of string [Activity state](_objects.md#activity-state) | optional | Whether to return only active, only deleted or both records. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of resource category image assignments returned. |
+
+### Response
+
+```javascript
+{
+    "ResourceCategoryImageAssignments":[
+        {
+            "Id": "edb5f556-7afb-4650-8d4e-8c0a6fff784d",        
+            "CategoryId": "5c0804f9-d03a-4b13-a57d-b00300781a41",
+            "ImageId": "e910d008-fded-4af5-a84f-c00f92e3947d",
+            "IsActive": true,
+            "CreatedUtc": "2023-10-01T11:48:57Z",
+            "UpdatedUtc": "2023-10-28T11:48:57Z"
+        },
+        {
+            "Id": "9d18f5fb-cce5-4e70-9561-f7804262344b",            
+            "CategoryId": "47d6b462-35ec-467e-a565-b00300781a41",
+            "ImageId": "11056cdb-2045-49e0-821f-2b93905ff522",
+            "IsActive": true,
+            "CreatedUtc": "2023-10-01T11:48:57Z",
+            "UpdatedUtc": "2023-10-28T11:48:57Z"
+        }
+    ],
+    "Cursor": "28704948-77df-4bb4-8f39-f8380dc8a914"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ResourceCategoryImageAssignments` | array of [Resource category image assignment](resourcecategories.md#resource-category-image-assignment) | optional | Resource category image assignments. |
+| `Cursor` | string | required | Unique identifier of the last and hence oldest resource category image assignment returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older resource category image assignments. |
+
+#### Resource category assignment
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the assignment. |
+| `IsActive` | bool | required | Whether the assignment is still active. |
+| `CategoryId` | string | required | Unique identifier of the [Resource category](#resource-category). |
+| `ResourceId` | string | required | Unique identifier of the [Resource](#resource) assigned to the Resource category. |
+| `CreatedUtc` | string | required | Creation date and time of the assignment in UTC timezone in ISO 8601 format. |
+| `UpdatedUtc` | string | required | Last update date and time of the assignment in UTC timezone in ISO 8601 format. |
```

Comment:
This should be moved up to the "Get all resource category assignments" section.

Reply:
makes sense